.github/workflows/gh-actions-prover-benches.yml

```
@@ -0,0 +1,30 @@
+name: GH Actions Prover Benches on PR
+on:
+  pull_request:
+    types: [ labeled ]
```

```suggestion
  pull_request:
    types: [ labeled ,  ready_for_review]
```

On that way, we require the PR as `Ready for review` to be able to run the benches, So that WIP PRs do not get in.